src/logic/calculate.js

```diff
@@ -0,0 +1,8 @@
+import operate from './operate';
+
+const calculate = (data, buttonName) => {
+  const { total, next } = data;
```

Comment:
- Your data object should contain three properties, total, next and operation, as required in [the Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896082/)
![image](https://user-images.githubusercontent.com/46115725/73076497-d85f8d00-3e83-11ea-9b5c-22de6414b8f9.png)
- Also we are missing something very important as stated in [the Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896082/)
![image](https://user-images.githubusercontent.com/46115725/73076651-26749080-3e84-11ea-9c01-efaa2f11dd44.png)
- This means that you need to start working on the logic on how  your logic should behave given the user's input for example if your buttonName argument is `AC` then we should set `{total: null, next: null, operation: null}`, since that is the function of that particular button in the calculator to clear the app of all data, and you should follow suit with the rest of the operations, and inputs you might possibly get, this is going to be closely related to the next milestone where you are to connect this logic to the display of your calculator, so you will have to take into account the users input when there is no operation, so when the user is clicking a number, that should be accounted for, when the user presses an operation, when the user starts typing a number after the operation and so on. You will leave the operation logic to the operate file, so this part of the logic should be in charge of passing on to calculate the appropriate information.